DF-325-Removing redundant topics from the docs website PEM online help (#3787)
EnterpriseDB/docs: product_docs/docs/pem/9/pem_connecting_to_database_server.mdx

---
title: "Connecting and disconnecting a database server"
legacyRedirectsGenerated:
  # This list is generated by a script. If you need add entries, use the `legacyRedirects` key.
  - "/edb-docs/d/edb-postgres-enterprise-manager/reference/online-documentation/8.0/pem_connect_to_server.html"
# This file is moved from pem_online_help.
redirects:
- /pem/latest/pem_online_help/01_toc_pem_getting_started/09_pem_connect_to_server/
---

After defining a server connection, use the `Connect to Server` dialog to authenticate with a server and access the objects stored on the server. To access the dialog, right click on the server name in the PEM client tree control, and select `Connect Server` from the context menu.

If prompted, provide authentication information for the selected server:

> > -   Use the `Password` field to provide the password of the user that is associated with the defined server.

Reviewer: ```suggestion
-   Use the `Password` field to provide the user's password associated with the defined server.
```

> > -   Check the box next to `Save Password` to instruct the server to save the password for future connections; if you save the password, you will not be prompted when reconnecting to the database server with this server definition.

Reviewer: ```suggestion
-   Select the box next to `Save Password` to instruct the server to save the password for future connections. If you save the password, you are not prompted while reconnecting to the database server with this server definition.
```
